carbon-design-system/carbon
# Fixed icon password visibility in `FluidPasswordInput`

Closes #16314

Added classes to fix the button visibility postion when the `FluidPasswordInput` is Invalid.

#### Testing / Reviewing

- Go to `FluidPasswordInput`  and add an invalid prop to true.
- Make sure there is no regressions.
